group_vars/all.yml

```diff
@@ -51,3 +59,4 @@ nginx_enabled: true
 nginx_helm_chart_repo: "https://kubernetes.github.io/ingress-nginx"
 nginx_helm_chart_version: "4.5.2"
 nginx.prometheus_enabled: true
+nginx_oci_subnet_id: ""
```

Comment:
Lets add a comment to this explaining what this field is for.  Nothing too crazy, but at least mention that it is only needed when using the OCI platform.